src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -113,6 +120,37 @@ def init_db():
 
     return db_conn
 
+def init_mgmt_interface_tables(db_conn):
+    """
+    Initializes interface maps for mgmt ports
+    :param db_conn: db connector
+    :return: tuple of mgmt name to oid map and mgmt name to alias map
+    """
+
+    db_conn.connect(APPL_DB)
```

Comment:
>APPL_DB [](start = 20, length = 7)

This is not true according to the design doc. We decide to to use configDB and stateDB.